CHANGELOG.rst

```diff
@@ -5,11 +5,46 @@ Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
-[Unreleased]
-************
+Unreleased
+**********
+
+`1.0.0`_ (2021-07-22)
+*********************
 
 Added
 -----
 
+* Documentation
+  * Example notebooks
+  * Setup guide (`#120 <https://github.com/eWaterCycle/ewatercycle/issues/120>`_)
+  * HPC cluster guide
```

Comment:
The second level bullet items don't render correctly, see https://github.com/eWaterCycle/ewatercycle/blob/changelog-1.0.0/CHANGELOG.rst. I think we should use a different syntax like `-`.

Reply:
added newlines to fix